setting-up-github-repositories.md

```diff
@@ -0,0 +1,53 @@
+# Setting up github repositories for specifications
+
+This document tries to describe how we set up github repositories
+for specifications or specification-like documents.
+These documents are generally maintained in either
+[bikeshed](https://github.com/tabatkins/bikeshed/)
+or
+[respec](https://github.com/w3c/respec/).
+
+Also see [W3C's FAQ on using GitHub](https://w3c.github.io/faq.html).
+
+## Structuring the repository
+
+The primary branch of the repository should be `master`, which is `git`'s default.  (We should not manually interact with a `gh-pages` branch, although some older repositories are set up to do so.)
```

Comment:
I don't think we should recommend using the name `master` for the primary branch, despite it being `git`'s default.

Reply:
I hope I've addressed your concern in https://github.com/w3ctag/process/pull/5/commits/0430f1c168f2be4fcc5488a1671e8d273c24ed60 -- but perhaps you could double-check?

Reply:
Looks good.